### PR TITLE
[cinder]Remove dependence of volumes on the snapshot it was created from

### PR DESCRIPTION
**Pull request by sjamgade**
By default snapshots cannot be deleted if the volume it created from
that snapshot is still present.

### CODE REVIEW

File: scripts/openstack-quickstart-demosetup [hunk 1]
```diff
@@ -350,6 +350,7 @@ crudini --set $c $cinder_backend_name volume_driver "cinder.volume.drivers.lvm.L
 crudini --set $c $cinder_backend_name volume_group "cinder-volumes"
 crudini --set $c $cinder_backend_name lvm_type "thin"
 crudini --set $c $cinder_backend_name iscsi_protocol "iscsi"
+crudini --set $c $cinder_backend_name rbd_flatten_volume_from_snapshot true
```
Comment (toabctl) [inline scripts/openstack-quickstart-demosetup]: Why is that needed? We do not configure a rbd backend in quickstart. Or am I missing something?

**Comment by sjamgade** [comment]
oops sorry, this is supposed to go in mkcloud and not on quickstart !
I thought jenkins-tempest-full-job is cleanvm !
